winit/src/application.rs

```diff
@@ -615,12 +615,25 @@ pub fn run_command<A, E>(
                 }
             },
             command::Action::Window(action) => match action {
+                window::Action::Drag => {
+                    let _res = window.drag_window();
+                }
                 window::Action::Resize { width, height } => {
                     window.set_inner_size(winit::dpi::LogicalSize {
                         width,
                         height,
                     });
                 }
+                window::Action::ResizeDrag => {
+                    // TODO: Requires patch to winit
+                    // let _res = window.drag_resize_window();
+                }
+                window::Action::Maximize => {
+                    window.set_maximized(!window.is_maximized());
+                }
+                window::Action::Minimize => {
+                    window.set_minimized(true);
+                }
```

Comment:
Does it make sense to take a bool here? `Action::Minimize(bool)`

Reply:
From what I've read, only X11 supports setting false, but yeah it could be supported.

Reply:
Looking at the implementation in winit and at https://docs.rs/winit/latest/winit/window/struct.Window.html#method.set_minimized, `set_minimized(false)` works on Windows, macOS, and X, but not Wayland.